cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java

```diff
@@ -205,10 +205,10 @@ public void testInnerJoin() throws Exception {
     DataSetManager<Table> sinkManager = getDataset("joinerOutput");
     Set<StructuredRecord> expected = ImmutableSet.of(joinRecordSamuel, joinRecordJane);
     Set<StructuredRecord> actual = Sets.newHashSet(MockSink.readOutput(sinkManager));
-    Assert.assertEquals(expected, actual);
-
-    validateMetric(2, appId, "testJoiner.records.out");
-    validateMetric(2, appId, "sink1.records.in");
```

Comment:
Put a TODO and jira id (i guess there is an issue for this flaky test) here to enable it.